.gitmodules

```diff
@@ -1,7 +1,9 @@
 [submodule "spack"]
   path = spack
-  url = https://github.com/jcsda/spack
-  branch = release/1.9.0
+#  url = https://github.com/jcsda/spack
+#  branch = release/1.9.0
+  url = https://github.com/AlexanderRichert-NOAA/spack
+  branch = g2c_update_feb25
```

Comment:
This branch doesn't exist? All I found were the following. I assume it did exist when you made this PR.

* [g2c_develop](https://github.com/AlexanderRichert-NOAA/spack/tree/g2c_develop)
* [g2c_update_apr24](https://github.com/AlexanderRichert-NOAA/spack/tree/g2c_update_apr24)
* [g2c_update_july2023](https://github.com/AlexanderRichert-NOAA/spack/tree/g2c_update_july2023)
* [g2c_updates_jul24](https://github.com/AlexanderRichert-NOAA/spack/tree/g2c_updates_jul24)

Reply:
I need to revert it to the regular submodule now that those changes are merged...